src/main.js

```diff
@@ -230,12 +207,45 @@ function switchScreens(mainView, alternateView) {
   alternateView.classList.toggle('hidden')
 }
 
-// function posterFormToggle() {
-//   document.querySelector('.main-poster').classList.toggle('hidden')
-//   document.querySelector('.poster-form').classList.toggle('hidden')
-// }
+// created new function for adding event listeners to each mini poster and
+// for deleting mini posters
+function deleteMiniPoster() {
+  var miniPosters = document.querySelectorAll('.mini-poster')
+
+  for (var i = 0; i < miniPosters.length; i++) {
+    miniPosters[i].addEventListener('dblclick', function() {
+      var thisPosterID
+
+      if (event.target.id){
+        thisPosterID = event.target.id
+      } else {
+        thisPosterID = event.target.parentElement.id
+      }
 
-// function savedPostersToggle() {
-//   document.querySelector('.main-poster').classList.toggle('hidden')
-//   document.querySelector('.saved-posters').classList.toggle('hidden')
-// }
+      for (var i = 0; i < savedPosters.length; i++) {
+        if (savedPosters[i].id == thisPosterID){
+          savedPosters.splice(i, 1)
+        }
+      }
+
+      if (event.target.classList === 'mini-poster') {
+        event.target.classList.add('hidden')
+      } else {
+        event.target.parentElement.classList.add('hidden')
+      }
+    })
+  }
+}
+
+// created this function to ensure that all input fields have information in
+// them before allowing a poster to be created
+function validateForm(userImageURL, userTitle, userQuote) {
+  if (!userImageURL || !userTitle || !userQuote) {
+    alert('All fields must be filled out')
+    return false
+  } else if (!userImageURL.includes('jpeg') && !userImageURL.includes('png') && !userImageURL.includes('gif') && !userImageURL.includes('jpg')) {
```

Comment:
Why && instead of || here? Maybe I'm misunderstanding what this is doing?

Reply:
We wanted to only fail the test if the did not include any of the strings. We basically wanted to make sure that the string didn't include x and also didn't include y, z, or q.. if that makes sense. I believe the && is correct in this case rather than || but we can discuss further if the reasoning is unclear!